ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/DbDispatcher.scala

```diff
@@ -15,17 +15,17 @@ import com.google.common.util.concurrent.ThreadFactoryBuilder
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.control.NonFatal
 
-final class DbDispatcher(
+final class DbDispatcher private (
     val maxConnections: Int,
     connectionProvider: HikariJdbcConnectionProvider,
-    sqlExecutor: ExecutorService,
+    executor: Executor,
     metrics: MetricRegistry,
 )(implicit logCtx: LoggingContext)
     extends ReportsHealth {
 
   private val logger = ContextualizedLogger.get(this.getClass)
 
-  private val sqlExecution = ExecutionContext.fromExecutorService(sqlExecutor)
+  private val executionContext = ExecutionContext.fromExecutor(executor)
```

Comment:
I think it would be wise to keep the naming reflect what the executor will be used for.

Reply:
I agree, but I don't think the original name was great. How about `transactionExecutionContext`?

Reply:
Sure.

Reply:
Reverted the file, with only a couple of small changes.